kanidm_tools/src/cli/account.rs

```diff
@@ -281,6 +283,35 @@ impl AccountOpt {
                         }
                     }
                 }
+                AccountCredential::GenerateBackupCode(acsopt) => {
+                    let client = acsopt.copt.to_client();
+                    match client.idm_account_primary_credential_generate_backup_code(
+                        acsopt.aopts.account_id.as_str(),
+                    ) {
+                        Ok(s) => {
+                            println!("GenerateBackupCode success.");
```

Comment:
Probably need some instructions that these codes should be saved for future use.